### PR TITLE
Fix display of enums for nullable property

### PR DESCRIPTION
**Pull request by AirBair**
## Subject

When displaying an enum in list or show view, the twig template don't check for potential null value (nullable property in entity for example) and crash with `Impossible to access an attribute ("name") on a null variable.`

This fix just add the missing check in `display_enum.html.twig`.

I am targeting this branch, because this is a patch without breaking change.

## Changelog

```markdown
### Fixed
- Fix display of enums for nullable property
```

### CODE REVIEW

**Review by bedfir — APPROVED** [review]

**Review by VincentLanglet — APPROVED** [review]

File: src/Resources/views/CRUD/display_enum.html.twig [hunk 1]
```diff
@@ -10,5 +10,9 @@ file that was distributed with this source code.
 #}
 
 {%- apply spaceless %}
-    {{ value.name }}
+    {%- if value is null -%}
+        &nbsp;
+    {%- else -%}
```
Comment (jordisala1991) [inline src/Resources/views/CRUD/display_enum.html.twig:15]: isnt it better to just add an if? the else part where it is null can be omitted.

Comment (AirBair) [reply src/Resources/views/CRUD/display_enum.html.twig:15]: I've follow what has been done in similar template (currency, percent, etc..) for coherency, but i can make the change requested, your call!

Comment (VincentLanglet) [reply src/Resources/views/CRUD/display_enum.html.twig:15]: I saw that all the other display_ template were displaying an nbsp when the value was empty @jordisala1991

**Review by jordisala1991 — APPROVED** [review]

**Comment by VincentLanglet** [comment]
Thanks

**Pull request merged**